styles.css

```diff
@@ -10,7 +10,7 @@
   border-radius: 4px !important;
 }
 
-.DashboardProfileCard-avatarImage, .ProfileAvatar-image, .ProfileAvatar {
+.DashboardProfileCard-avatarImage, .ProfileCard-avatarImage, .ProfileAvatar-image, .ProfileAvatar {
```

Comment:
Looks like this is halfway there! It wasn't fully working so upon inspection I noticed there is also a `.ProfileCard-avatarLink` class that needs to be targeted in addition to the `.ProfileCard-avatarImage` you added. If you add that class as well, this can be merged!

Reply:
I couldn't reproduce the bug. I see there is a surrounding `.ProfileCard-avatarLink` also with originally 50% radius but everything was working for me without touching it. It didnt break anything to include the class though, so Im gonna do as you say

BTW, congrats for the extension, twitter looks so much better this way